experimenter/experimenter/experiments/constants.py

```diff
@@ -763,3 +763,21 @@ class EmailType(models.TextChoices):
     ERROR_FEATURE_TARGET_COLLECTION = (
         "Feature '{feature_id}' publishes to collection '{collection}'"
     )
+
+
+RISK_QUESTIONS = {
+    "BRAND": (
+        "If the public, users or press, were to discover this experiment and "
+        "description, do you think it could negatively impact their perception "
+        "of the brand?"
+    ),
+    "MESSAGE": ("Does your experiment include ANY messages? If yes, this requires the"),
```

Comment:
This is cut off.

Reply:
Yes, I copy pasted from here- https://github.com/mozilla/experimenter/blob/208335e19f2d25f09001304dcc31e4f9c198ada8/experimenter/experimenter/nimbus-ui/src/lib/constants.ts#L83 and this thing I will fix this when I will be dealing with all the links, but if you see the screenshot it says `message consult` to complete the sentence